spec/services/catalog/soft_delete_restore_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe Catalog::SoftDeleteRestore do
+  let(:tenant) { create(:tenant) }
+  let(:portfolio_item) { create(:portfolio_item, :tenant_id => tenant.id, :discarded_at => Time.current) }
+
+  context "when restoring a soft-deleted record" do
```

Comment:
More nitpicking, obviously the specs will still work, but there should be an outer `describe ".process" do` block.